src/task2.py

```diff
@@ -0,0 +1,19 @@
+"""Tuple practice"""
+# 1
+lst = ['a', 'b', 'c']
+tpl = tuple(lst)
+
+# 2
+tpl = ('a', 'b', 'c')
+lst = list(tpl)
+
+print("{}: {}".format(type(lst), lst))
```

Comment:
why do we have this prints but print nothing above? it seems it should be similar